pkg/portforward/portforward.go

```diff
@@ -0,0 +1,207 @@
+package portforward
+
+// based on https://github.com/justinbarrick/go-k8s-portforward
```

Comment:
"licensed under the Apache License 2.0, and adapted in the following ways:
..."

(the latter bit only if you changed anything significant)

Reply:
I changed only the tests